microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -21,33 +21,29 @@ ms.date: 09/13/2023
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# Edit files with Workspaces (Filesystem tab)
+# Edit and save files in a workspace
+<!-- https://developer.chrome.com/docs/devtools/workspaces -->
 
-Use the **Filesystem** tab in the **Sources** tool to define a workspace, to save DevTools changes in your source code files rather than only in a transient copy of the files that are returned by the web server.
+Use the **Workspace** tab in the **Sources** tool to define a workspace, to save DevTools changes in your source code files rather than only in a transient copy of the files that are returned by the web server.
 
-This tutorial provides hands-on practice in setting up and using a workspace in DevTools. After you add files to a workspace, changes that you make in your source code by using DevTools are saved on your local computer. These changes are preserved across page refreshes.
+This tutorial provides hands-on practice in setting up and using a workspace in DevTools.  After you add files to a workspace, changes that you make in your source code by using DevTools are saved on your local computer.  These changes are preserved across page refreshes.
 
-To refresh your knowledge of the technologies used, see the following articles:
-
-* Use HTML, CSS, and JavaScript to build a web page. See [Getting started with the web](https://developer.mozilla.org/docs/Learn/Getting_started_with_the_web)
-
-* Use DevTools to make basic changes to CSS. See [Get started viewing and changing CSS](../css/index.md).
-
-* Run a local HTTP web server. See: 
-
-  * Using Node.js (used for this tutorial): [Set up a localhost server](../../visual-studio-code/microsoft-edge-devtools-extension/install.md#step-6-set-up-a-localhost-server) in _Installing the DevTools extension for Visual Studio Code_.
-  
-  * Using Python: [Running a simple local HTTP server](https://developer.mozilla.org/docs/Learn/Common_questions/set_up_a_local_testing_server#running_a_simple_local_http_server) in _How do you set up a local testing server?_
+See also:
+* [Getting started with the web](https://developer.mozilla.org/docs/Learn/Getting_started_with_the_web) - How to use HTML, CSS, and JavaScript to build a web page. 
```

Comment:
Not sure why we link to Getting started with the web here. Knowing what the web is, and knowing about HTML, CSS, and JS is kind of a prereq for the entire DevTools, not just the Workspace tab. So, either we tell that to readers in our overview article only, or we mention it everywhere. But mentioning it only here feels odd to me.

Reply:
inspecting doc design: upstream vs. downstream PR vs. what it should be:
upstream: https://developer.chrome.com/docs/devtools/workspaces
starts w/ 
![image](https://github.com/user-attachments/assets/db20bc2d-8775-40c3-944d-b364c21d7c93)
* https://developer.mozilla.org/docs/Learn/Getting_started_with_the_web (link text mismatch/outdated, latest dest title is "Your first website")
* https://developer.chrome.com/docs/devtools/css
* https://developer.mozilla.org/docs/Learn/Common_questions/set_up_a_local_testing_server#Running_a_simple_local_HTTP_server

@captainbrosset

Reply:
since I have made this Workspace tut standalone and targeted all user levels (Maint'ers) w/ end-to-end steps spelled out, that reduces any need to link-away.  i'll at least move the 3 links to See Also at bottom, or move them to an Overview article.